apps/task/filters.py

```diff
@@ -42,9 +42,15 @@ class Meta:
 
 
 class TimeEntryFilter(django_filters.FilterSet):
+    date_lte = django_filters.CharFilter(
+                     field_name='date',
+                     lookup_expr='lte'
+    )
+    date_gte = django_filters.CharFilter(
+                     field_name='date',
+                     lookup_expr='gte'
+    )
+
```

Comment:
Let's use standard indentation.
```suggestion
    date_lte = django_filters.CharFilter(
        field_name='date',
        lookup_expr='lte'
    )
    date_gte = django_filters.CharFilter(
        field_name='date',
        lookup_expr='gte'
    )
```